src/content/reference/domain-allowlist/index.md

```diff
@@ -57,10 +57,18 @@ Below is a list of the external domains we require access to for our clusters to
     - domains:
         - `*.keybase.io`
     - Vault initialisation and unsealing requires access to Keybase.
+- letsencrypt.org
+    - domains:
+        - `*.api.letsencrypt.org`
+    - cert-manager will request certificates from Lets Encrypt.
 - quay.io
     - domains:
         - `*.quay.io`
     - Container images are hosted on Quay.
+- sentry.io
+    - domains:
+        - `o346224.ingest.sentry.io`
```

Comment:
Is this still used? never seen it but maybe I am just not aware

Reply:
yeah i saw happa trying to reach out to it in bumblebee earlier